packages/lambda/preset.js

```diff
@@ -10,7 +10,30 @@ module.exports = {
     stageName: 'prod',
     uniqueName: 'hops-lambda-<name>',
     cloudformationTemplateFile: path.resolve(__dirname, 'cloudformation.yaml'),
-    include: ['<cacheDir>/**'],
+    include: ['<serverDir>/**'],
```

Comment:
Out of curiosity: could this be a breaking change for someone?

Reply:
I don't think so, it's a mistake on our side, because we removed `cacheDir` and added `serverDir` instead in untool.
And we have a workaround already in place in the `aws-config.js` file:
https://github.com/xing/hops/blob/doctor/packages/lambda/lib/aws-config.js#L23

Reply:
I removed the workaround in the `aws-config.js` file.